### ISSUE TITLE
Logo should have all of its details preserved in dark forced-color themes

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail.io#303]
Follow-up to #295. Our logotype is visible in high-contrast dark themes, but the details of the bird are lost. The logo’s full details should be visible across all contrast themes.

**Comment by Scotchester** [comment]
Go for it, @satvikvash!

**Comment by thibaudcolas** [comment]
Copying what @satvikvash and I discussed in DMs – here are the steps needed to get the site set up enough to work on this:

1. `npm install && npm run build`
2. createsuperuser
3. Log into the CMS
4. Delete the default homepage
5. Create a new Homepage
6. Create a new Site and assign it the homepage

**Comment by dassudip2001** [comment]
it is still pending or fixes it ??

**Comment by vossisboss** [comment]
@thibaudcolas Any recommendations on how to emulate this issues on non-Windows machines? @Stormheg is poking around BrowserStack, but it looks like we need a smidge of help recreating the issue. It looks like we've updated the logo since this issue was filed as well, so we need to do some fresh testing.

**Comment by thibaudcolas** [comment]
I can confirm this is still an issue. There are two options if you want to try it out yourself:

1. In a Chromium browser, in the DevTools’ Rendering panel, set "Emulate CSS media feature forced-colors" to "forced-colors: active"
2.. Ask our accessibility team for access to our shared [Assistiv Labs](https://assistivlabs.com/) account, which supports trying out this theming as it’s implemented in Windows

**Comment by vossisboss** [comment]
Thank you for the additional context and testing suggestions Thibaud!
